### ISSUE TITLE
Questions about some training details and benchmark of Depth Anything v2

### ISSUE THREAD

**Issue by omega-cw** [LiheYoung/Depth-Anything#198]
Hi, wonderful work. And I have a few questions about training details and benchmark hope you can help with me :
(1) what is the loss function for training teacher model and student model? 
(2) Is there any other loss besides scale- and shift-invariant loss L_{ssi} for teaching model?
(3) Is scale- and shift-invariant loss L_{ssi} used for the training of student model?

(4) About benchmark. What's the criterion of agree and disagree when voting with the predicted results by four expert models?

Sincerely looking forward to your reply. Thanks in advance!!

**Comment by LiheYoung** [comment]
Hi,
(1) Both teacher and student use two objectives: scale-shift invariant loss and gradient matching loss.
(2) We also add the gradient matching loss for all our models, as highlighted in the paper.
(3) Yes.
(4) For a pair of points, if the relative depth (i.e., which point is closer) judged by the four models is different (as long as one model is different from others), this is a disagreement.

**Comment by omega-cw** [comment]
> Hi, (1) Both teacher and student use two objectives: scale-shift invariant loss and gradient matching loss. (2) We also add the gradient matching loss for all our models, as highlighted in the paper. (3) Yes. (4) For a pair of points, if the relative depth (i.e., which point is closer) judged by the four models is different (as long as one model is different from others), this is a disagreement.

Great appreciation for your reply!
